packages/zoltan2/test/temp/vtxlabel.cpp

```diff
@@ -0,0 +1,260 @@
+#include "Tpetra_Core.hpp"
```

Comment:
You can delete this test.  femv.cpp does all the same and more (and has better comments).  Thanks.

Reply:
Done.